tests/server/index.js

```diff
@@ -96,6 +96,16 @@ tape('Test the submit message page', (t) => {
   })
 })
 
+tape('Test for random startups page', (t) => {
+  supertest(server).get('/randomStartups').end((err,res) => {
+    t.error(err, 'No Error');
+    var testedClass = '<div class="dib mh1 tc">';
+    var count = (res.text.match(/<div class="dib mh1 tc">/g) || []).length;
```

Comment:
you could change to those to const to keep it consistant

Reply:
you're right